Makefile

```diff
@@ -27,8 +27,10 @@ build::
 	done
 	cd ${PACKDIR}/nodejs/ && \
 		yarn install && \
+		# pick up the local versions of these if possible.
+		(yarn link @pulumi/pulumi || true) && \
 		yarn run tsc && \
-		cp ../../README.md ../../LICENSE package.json yarn.lock ./bin/ && \
```

Comment:
it's unclear to me if we should be including yarn.lock's in our libraries.  yarn.lock should be locking your dependencies at the application level.  however, libraries should just say what their constraints are in their package.json.  Then npm/yarn should figure out what to do at the application level with all of this.